### DISCUSSION TITLE
Error: Unable to find a id in "LM"

### DISCUSSION THREAD

**Discussion by oscarmbv** [pear-devs/pear-desktop#1019] [General]
Hello, apologies if this has been already asked. I'm trying to download a autogenerated list (liked songs), and I'm getting this error Unable to find a id in "LM"
I tried another regular list and it is working fine, perhaps it is not possible to download autogenerated lists or I missing something?
Thank you in advance

**Comment by Araxeus** [comment]
Yup you cant download auto-generated playlists

You can only download public playlists
